docs/markdown/Contributing.md

```diff
@@ -208,6 +210,13 @@ $ cd meson
 $ flake8
 ```
 
+To run it automatically before committing:
+
+```console
+$ echo flake8 >>.git/hooks/pre-commit
+$ chmod +x .git/hooks/pre-commit
```

Comment:
`flake8` takes a very long time to run for me on the entire codebase, is there a way to make it only check for issues in the files touched by the commit?

Reply:
Apparently, `flake8 --install-hook=git` writes a hook which does that.